src/decision/subroutines/qualification/include/GoThroughGate.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Created By: Reid Oliveira
+ * Created On: May 19, 2018
+ * Description:
```

Comment:
We should fill in the "Description" here and in the `.cpp` file

Reply:
done